package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "standard --fix | snazzy"
   },
   "dependencies": {
-    "@nearform/clinic-common": "^2.0.0",
+    "@nearform/clinic-common": "nearform/node-clinic-common#separate-data",
```

Comment:
Is this correct?

Reply:
Temporary until https://github.com/nearform/node-clinic-common/pull/21 is released, will then change to `^2.4.0` before merging :)